Changelogs for Digital Twin Manager Plugins 1.2.0 (#2598)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/dtm-1.2.0-added-asset-properties-widget-into-dtm-plugins-component.md

---
date: ""

> title: "Added asset properties widget into DTM plugins component"

Reviewer: ```suggestion
title: "Added Asset properties widget into DTM plugins component"
```

product_area: "Application enablement & solutions"
change_type:
    - value: "change-QHu1GdukP"
      label: "Feature"
component:
    - value: "component-Tl88RYb4A"
      label: "Digital Twin Manager"
build_artifact:
    - value: "tc-wYIY0MBDO"
      label: "dtm"
ticket: "CTM-1625"
version: "1.2.0"
---

> The DTM plugins component now includes a new Asset Properties widget module, which enhances the functionality of the existing Asset Properties widget.

Reviewer: ```suggestion
The DTM plugins component now includes a new "Asset properties" widget module, which enhances the functionality of the existing "Asset properties" widget. This includes:
```

> The DTM plugins component now includes a new "Asset properties" widget module, which enhances the functionality of the existing "Asset properties" widget. This includes:

Reviewer: ```suggestion
The DTM plugins component now includes a new asset properties widget module, which enhances the functionality of the existing "Asset properties" widget. This includes:
```

>  - Simplified selection of the modeled properties for assets

Reviewer: ```suggestion
 - Simplified selection of the modelled properties for assets.
```

>  - Support for listing properties of type : file

Reviewer: ```suggestion
 - Support for listing properties of type: file
```

>  - Ability to preview images attached to property type: file 

Reviewer: ```suggestion
 - Ability to preview images attached to properties of type: file 
```

 - The widget has been migrated to Angular.

> The module is designed for easy integration into any {{< product-c8y >}} application. In future releases, this widget will replace the current Properties widget across all {{< product-c8y >}} applications.

Reviewer: ```suggestion
The module is designed for easy integration into any {{< product-c8y >}} application. In future releases, this widget will replace the current "Asset properties" widget across all {{< product-c8y >}} applications.
```
